specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -69,53 +69,53 @@ paths:
           schema:
             type: "string"
         - in: query
-          name: appartementsrechtvolgnummer
+          name: kadastraleaanduiding__appartementsrechtvolgnummer
           description: "Nummer dat het kadastraal object uniek identificeert als een\
             \ appartementsrecht binnen het complex."
           schema:
             type: "integer"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
           description: "Het burgerservicenummer van de persoon die een zakelijk recht op een kadastraal onroerende zaak heeft. Deze persoon is zakelijk gerechtigde van de kadastraal onroerende zaak. Door deze query-parameter te gebruiken worden Kadastraal Onroerende Zaken geretourneerd waar deze persoon een zakelijk recht op heeft."
           required: false
           schema:
             type: string
             maxLength: 9
         - in: query
-          name: kadasterpersoonidentificatie
+          name: persoon_kadasterpersoonidentificatie
```

Comment:
zie vorige opmerking, over burgerservicenummer. Dit zou ook zakelijkgerechtigden__persoon__identificatie worden.

Maar zakelijkgerechtigde__kadasterpersoonidentificatie lijkt mij duidelijker, om zo het onderscheid tussen ingeschrevenpersoon en kadasterpersoon aan te geven.

Probleem met deze properties is dat we zoeken op een gegeven dat geen deel uitmaakt van de resource zelf. Plus dat het gegeven in de gerelateerde resource niet op het hoofdniveau staat (2e laag). Plus dat we nu in de zakelijkgerechtigde de kadasterpersoon en ingeschrevenpersoon hebben samengevoegd tot één ding (persoon). Plus dat we daarin ook het burgerservicenummer stoppen in "identificatie".

Dit alles zou er denk ik voor pleiten ook bij het zoeken maar één pad voor alle personen te gebruiken. Alleen is voor gebruikers burgerservicenummer zo veel duidelijker dan "identificatie".

Reply:
Ik onderschrijf jouw probleemschets en kan meegaan met jouw voorstel. Maar je voorbehoud is ook niet onbelangrijk. Dat laatste was voor mij aanleiding om de voorgestelde prefixes te kiezen.